packages/core/src/components/slider/sliderHandle.tsx

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2018 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the terms of the LICENSE file distributed with this project.
+ */
+
+import { Intent } from "../../common";
+import { AbstractPureComponent } from "../../common/abstractPureComponent";
+
+export type SliderHandleType = "full" | "start" | "end";
+
+export interface ISliderHandleProps {
+    value: number;
+    trackIntentAfter?: Intent;
+    trackIntentBefore?: Intent;
+    type?: SliderHandleType;
+}
+
+export class SliderHandle extends AbstractPureComponent<ISliderHandleProps> {
```

Comment:
can just use `React.PureComponent` here as you're not using any of the `APC` features